wandb/integration/tensorboard/monkeypatch.py

```diff
@@ -63,13 +65,23 @@ def new_csfw_func(*args, **kwargs):
         )
         logdir_hist.append(logdir)
         root_logdir_arg = root_logdir
+        if root_logdir is None:
+            root_logdir_arg = logdir
 
         if len(set(logdir_hist)) > 1 and root_logdir is None:
             wandb.termwarn(
                 'When using several event log directories, please call wandb.tensorboard.patch(root_logdir="...") before wandb.init'
             )
-
-        if root_logdir is not None and not os.path.abspath(logdir).startswith(
+        # if the logdir containts the hostname, the writer was not given a logdir. In this case, the generated logdir
+        # is genetered and ends with the hostname, update the root_logdir to match.
+        hostname = socket.gethostname()
+        search = re.search(hostname, logdir)
+        if search:
+            wandb.termwarn(
+                "Experiment writer logdir is likely not given, updating root_logdir to match generated logdir."
+            )
```

Comment:
Let's just get rid of this message, it's more likely to confuse users than to help.